ghcide/src/Development/IDE/Spans/Documentation.hs

```diff
@@ -72,16 +72,14 @@ getDocumentationsTryGhc env mod names = do
   res <- fun
   case res of
       Left _    -> return []
-      Right res -> zipWithM unwrap (fmap snd $ Map.toList res) names
```

Comment:
Since `(Name,` is now present in `unwrap` - as was said before - we arrived to that `getDocsBatch` that does not require running `zipWithM` on its result.